src/vmm/src/vmm_config/mmds.rs

```diff
@@ -0,0 +1,20 @@
+// Copyright 2018 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+// SPDX-License-Identifier: Apache-2.0
+
+use std::net::Ipv4Addr;
+
+/// This struct represents the configuration realted to the MMDS service.
```

Comment:
```suggestion
/// Keeps the IPv4 address that can be used for the MMDS configuration. 
```

Reply:
Updated with: `/// Keeps the MMDS configuration`.